Makefile

```diff
@@ -0,0 +1,19 @@
+test:
+	nvim --headless -u ./init.lua -c "PlenaryBustedDirectory tests/ { minimal_init = './init.lua' }"
+#	nvim --headless -u tests/minimal_init.lua -c "PlenaryBustedDirectory tests/ { minimal_init = './tests/minimal_init.lua' }"
+
+lint: lint-lua lint-sh
+
+lint-lua:
+	pre-commit run --all-files luacheck
```

Comment:
I think pre-commit should rely on the Makefile more than the other way around

Reply:
I was just being lazy 😅